crates/workspace/src/pane.rs

```diff
@@ -2837,14 +2894,9 @@ impl Pane {
 
         self.items
             .iter()
-            .map(|item| item.item_id())
-            .filter(|item_id| {
-                if let Some(ix) = self.index_for_item_id(*item_id) {
```

Comment:
Additional:

Seems like redundant logic as `index_for_item_id` goes through same `self.items` to find index. Can use `enumerate` instead.